modules/ROOT/pages/automated-policy-apply.adoc

```diff
@@ -5,6 +5,20 @@ endif::[]
 
 Users with *Organization Admin*, *Environment Admin*, or with a role to *Manage Policies* in the environment can create, edit and delete Automated Policies.
 
+== Task Prerequisites
+
+Before you apply an automated policy, ensure that:
+
+* Mule runtime engine (Mule) is installed with xref:api-gateway-capabilities-mule4.adoc[API gateway capabilities] enabled.
+* An xref:api-instance-landing-page.adoc[API], configured with a basic endpoint or a proxy endpoint, exists in API Manager.
```

Comment:
I don't think you need the commas in this sentence.

Reply:
Done! Thanks!